iosxr_iso2vbox.py

```diff
@@ -386,6 +378,19 @@ def verboseprint(*args):
         run('VBoxManage export %s --output %s' % (vmname, ova_out))
         verboseprint('Created OVA %s' % ova_out)
 
+    # Run basic sanity tests
+    verboseprint('Testing Virtualbox')
+    iosxr_test_path = os.path.join(pathname, 'iosxr_test.py')
+    cmdstring = "python %s %s" % (iosxr_test_path, box_out)
+    verboseprint("Running: '%s'" % cmdstring)
+    result = (subprocess.check_output(cmdstring, shell=True))
```

Comment:
Since the test script is another python script, ideally you wouldn't need subprocess here - you should be able to `import` it and then call a `main()` or `run()` method. But that may not be readily feasible until you get this repackaged as a module.